sources/views/contacts.js

```diff
@@ -37,29 +37,36 @@ export default class Contact extends JetView {
 	}
 	init(view, url) {
 		this.list = this.$$("userList");
-		this.list.sync(contacts);
-		this.list.attachEvent("onAfterSelect", chosenId => {
-			const previousId = this.getParam("id");
-			const item = contacts.getItem(previousId);
+		contacts.waitData.then(() => {
+			this.list.sync(contacts);
```

Comment:
sync can be done outside of waitData, you do not need to wait for the data to perform this operation